.devcontainer/devcontainer.json

```diff
@@ -17,7 +17,7 @@
     "build": {
         "dockerfile": "Dockerfile",
         "args": {
-            "BUILD_VERSION": "0.6.02"
+            "BUILD_VERSION": "0.6.03"
```

Comment:
version updates should be done in a separate PR from the dockerfile update: dockerfile takes a while to update and if we commit both at the same time, the devcontainer would be broken for a while

Reply:
Removed from this PR